examples/getting_started/run.py

```diff
@@ -27,7 +27,10 @@
     config = spear.get_config(user_config_files=[os.path.realpath(os.path.join(os.path.dirname(__file__), "user_config.yaml"))])
 
     # create Env object
-    env = spear.Env(config)
+    if config.SIMULATION_CONTROLLER.AGENT == "SphereAgent":
+        env = spear.Env(config)
+    elif config.SIMULATION_CONTROLLER.AGENT == "WheeledVehicleAgent":
+        env = spear.OpenBotEnv(config)
```

Comment:
`OpenBotEnv` should live in the `getting_started` folder. I'm not convinced it should be located in `spear`.